spec/features/users_feature_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe User do
+
+  it 'allows the user to receive notifications' do
+    user = create(:user)
+    visit user_url(user.id)
+    expect(page).to have_content('Recieve mails for new activities')
+  end
+
+  it 'enables notification' do
+    user = create(:user)
+    visit user_url(user.id)
+    page.check('Receive_Mails_for_new_Activites')
+    click_link 'Save'
+    expect(page).to have_content('You will now receive Notifications.')
+  end
+
+  it 'disables notifications' do
+    user = create(:user)
+    byebug
+    visit user_url(user.id)
+    click_link 'Save'
+    expect(page).to have_content('You will now receive Notifications.')
```

Comment:
I think you expect here something different - like when I unchecked I should have message that I will not receive notifications